src/main/resources/avro/bdg.avdl

```diff
@@ -748,299 +748,304 @@ record Variant {
 }
 
 /**
- An enumeration that describes the allele that corresponds to a genotype.
+ Allele encodings for genotypes.
  */
-enum GenotypeAllele {
+enum Allele {
```

Comment:
I have tried to define the tables myself, and with the complexity involved in our nested objects, etc. it was a nightmare and I ended up putting it off for a while, especially for trying to just achieve compatibility with a single field. If we're expecting people to be able to use Impala with our schemas we need to keep this in mind.